scripts/asm-installer/install_asm

```diff
@@ -210,6 +211,7 @@ OPTIONS:
   -c|--ca                <CA>
 
   -o|--operator_overlay  <FILE PATH>
+  -r|--remote_overlay    <FILE PATH>
```

Comment:
Can we actually call this asm_overlay? After seeing the name, I first thought it could accept a URL to an overlay file.

Reply:
Not sure asm is the right choice, but remote is confusing.